tests/mocha/event_test.js

```diff
@@ -593,6 +593,7 @@ suite('Events', function() {
           oldXml: '<shadow xmlns="https://developers.google.com/blockly/xml"' +
               ' type="simple_test_block" id="testBlockId2"></shadow>',
           ids: [thisObj.shadowBlock.id], recordUndo: false})},
+      // TODO(4577) Test serialization of move event coordinate properties.
```

Comment:
I think you need `#4577` instead to get some tools to auto link to the issue (same below)

Reply:
Nice catch! Fixed.